python/ray/train/lightning/lightning_trainer.py

```diff
@@ -142,14 +144,19 @@ def fit_params(self, **kwargs) -> "LightningConfigBuilder":
         self._trainer_fit_params.update(**kwargs)
         return self
 
-    def ddp_strategy(self, **kwargs) -> "LightningConfigBuilder":
+    def strategy(self, name, **kwargs) -> "LightningConfigBuilder":
         """Set up the configurations of ``pytorch_lightning.strategies.DDPStrategy``.
 
         Args:
+            name: The name of your distributed strategy. You can choose
+                from "ddp" and "fsdp". Default: "ddp".
```

Comment:
Should we have a default as mentioned in the docstring?

```python
def strategy(self, name: str = "ddp", **kwargs):
```

Reply:
agree